lib/v2/screens/explore_screens/vote_screens/proposal_details/components/proposal_details_bottom.dart

```diff
@@ -20,6 +20,26 @@ class ProposalDetailsBottom extends StatelessWidget {
       builder: (context, state) {
         return Column(
           children: [
+            if (state.isAlreadyPrecasted)
+              Padding(
+                padding: const EdgeInsets.only(top: horizontalEdgePadding, left: horizontalEdgePadding),
+                child: Row(
+                  children: [
+                    RichText(
+                      text: TextSpan(
+                        children: [
+                          TextSpan(text: 'You have already', style: Theme.of(context).textTheme.subtitle2),
+                          TextSpan(
+                            text: ' Precast',
+                            style: Theme.of(context).textTheme.subtitle2!.copyWith(color: AppColors.green1),
```

Comment:
Sorry I forget this copyWith not sure if I must create a new style It's not in figma and I already asked.

Reply:
Style added.